apps/api/lib/api/web/controllers/user_controller.ex

```diff
@@ -1,6 +1,28 @@
 defmodule API.Web.UserController do
   use API.Web, :controller
 
+  def search(conn, %{"email" => email}) do
+    users = API.User.search_users(email: email)
+
+    conn
+    |> put_status(:ok)
+    |> render(%{users: users})
+  end
+
+  def search(conn, %{"username" => username}) do
+    users = API.User.search_users(username: username)
+    conn
+    |> put_status(:ok)
+    |> render(%{users: users})
+  end
+
+  def search(conn, _params) do
```

Comment:
My reasoning here is that this a pretty small check against a pretty common occurrence, where a request needs to provide query params for filtering but for some reason omits them. How does the action fallback handle this case?